libs/can/can.c

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 Olin Electric Motorsports
+ */
+#include "libs/can/api.h"
+#include "libs/can/mob.h"
+#include <avr/io.h>
+
+void can_init(baud_rate_t baud) {
```

Comment:
http://wiki.olinelectricmotorsports.com/display/EL21/Git+Revamp+V2

I don't feel like we need to distinguish between public and private things. If a function is private to a specific translation unit (i.e. file), it can be private by just declaring it `static`. Or we can just prefix it with `private_`